net/BigBuffers.JsonParsing/JsonParser.cs

```diff
@@ -541,8 +541,9 @@ public T Parse(JsonElement element)
         {
           while (DeferredActions.TryTake(out var filler))
             filler();
-
+#if DEBUG
           Placeholder.ValidateUnfilledCount(Builder, unfilledCount);
+#endif
```

Comment:
```suggestion
          Placeholder.ValidateUnfilledCount(Builder, unfilledCount);
```

Reply:
ValidateUnfilledCount is already marked `[Conditional("DEBUG")]` - doesn't need #if DEBUG around it.